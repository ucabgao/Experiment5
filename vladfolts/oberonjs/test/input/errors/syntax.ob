MODULE m1;
VAR i: INTEGER;
BEGIN
	i := 1;
	!!!
END m1.