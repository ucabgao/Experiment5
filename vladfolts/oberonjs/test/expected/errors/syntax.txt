line 5: END expected (MODULE)
